### ISSUE TITLE
[2.1.1] M291 Z1 - jog buttons missing 

### ISSUE THREAD

**Issue by bernhardberger** [chrishamm/DuetWebControl#228]
I have a macro for adjusting the Z-Offset, however on `M291 Z1` the jog buttons are missing since 2.1.1 (maybe even 2.1.0, not sure).

```gcode
M291 P"Lower the nozzle to until it touches a 0.10mm shim. Click to continue." S2 Z1
```

Expected:
Jog buttons being displayed in the dialog and an "OK" button.

Actual behaviour:
Only the "OK" button is showing up.

https://reprap.org/wiki/G-code#M291:_Display_message_and_optionally_wait_for_response

**Comment by chrishamm** [comment]
Thanks, it's a known issue which will be fixed in DWC 2.1.2.

**Comment by chrishamm** [comment]
Fixed in v2.1.2.
